### PR TITLE
Update draft-ietf-perc-private-media-framework.md

### PR DESCRIPTION
**Pull request by dabenham**
Other than revisiting section 3.1.1 and 5.3 after changes to other drafts are complete, these changes either directly, or in-spirit, address all of the feedback/comments.

### CODE REVIEW

File: private-media-framework/draft-ietf-perc-private-media-framework.md [hunk 1]
```diff
@@ -211,12 +214,13 @@ which this framework's key exchange mechanisms will prevent.
 An endpoint's ability to join a conference hosted by a Media Distributor **MUST NOT**
 alone be interpreted as being authorized to have access to the E2E media
 encryption keys, as the Media Distributor does not have the ability to determine
-whether an endpoint is authorized.
+whether an endpoint is authorized.  Trusted endpoint authorizion is described
```
Comment (paulej) [inline private-media-framework/draft-ietf-perc-private-media-framework.md]: authorizion <= typo

File: private-media-framework/draft-ietf-perc-private-media-framework.md [hunk 2]
```diff
@@ -536,7 +542,7 @@ convey the fingerprint information per [@RFC5763].  An endpoint's SIP
 User Agent would send an INVITE message containing SDP for the media
 session along with the endpoint's certificate fingerprint, which can be
 signed using the procedures described in [@RFC4474] for the benefit of
-forwarding the message to other entities.  Other entities can now verify
+forwarding the message to other entities by the Focus [@RFC353].  Other entities can now verify
```
Comment (paulej) [inline private-media-framework/draft-ietf-perc-private-media-framework.md]: RFC number is wrong

File: private-media-framework/draft-ietf-perc-private-media-framework.md (hunk 1 above)
Comment (dabenham) [inline private-media-framework/draft-ietf-perc-private-media-framework.md]: Yes, should be " authorization "

File: private-media-framework/draft-ietf-perc-private-media-framework.md (hunk 2 above)
Comment (dabenham) [inline private-media-framework/draft-ietf-perc-private-media-framework.md]: Yeap, left out the 4 accidentally.   Should be RFC4353 .

**Pull request merged**